src/packages/documents/documents/workspace/views/info/document-workspace-view-info-history.element.ts

```diff
@@ -88,6 +90,17 @@ export class UmbDocumentWorkspaceViewInfoHistoryElement extends UmbLitElement {
 		this.#getLogs();
 	}
 
+	#onRollbackModalOpen = async () => {
+		const modalManagerContext = await this.getContext(UMB_MODAL_MANAGER_CONTEXT);
+		const modalContext = modalManagerContext.open(this, UMB_ROLLBACK_MODAL, {});
+
+		if (!modalContext) return;
+
+		modalContext.onSubmit().then(() => {
+			console.log('Rollback modal submitted');
```

Comment:
I'm not sure it's a good idea to have console logs here. Should we show a notification or maybe it doesn't matter?